NOTICE.BINARY

```diff
@@ -502,216 +293,9 @@ including, but not limited to:
  - Apache Commons
  - Apache Xerces
 
-ICU4J, (under analysis/icu) is licensed under an MIT styles license
```

Comment:
In the Derby section above, I think we can remove the section pertaining to Lucene jars, I don't think we have those in the project

Reply:
Thanks, removed.